RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  CustomerCenterView.swift
+//
+//
+//  Created by Andrés Boedo on 5/3/24.
+//
+
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct CustomerCenterView: View {
+
+    @StateObject private var viewModel = CustomerCenterViewModel()
```

Comment:
is a viewmodel overkill?

Reply:
Maybe, seems pretty light on logic. But I'm not opposed to it, even less if it's consistent with other screens.

Reply:
I think it's helpful.

Reply:
I will just keep it since it's already done haha. It is the first one I did and then I was like meh, could have been a `@State`. I am not that proficient with SwiftUI and wasn't sure, we should probably write up somewhere some guidelines on this :)